src/oasis-nexus/api.ts

```diff
@@ -145,21 +145,26 @@ declare module './generated/api' {
 
 export const isAccountEmpty = (account: RuntimeAccount | Account) => {
   if (account.layer === Layer.consensus) {
-    // TODO: find a sane way to recognize an important consensus account.
-    // The heuristics below is clearly insufficient, because it would indicate even named accounts like "Governance Escrow" to be empty.
-    return false
-    // const { available, size, nonce, debonding_delegations_balance, delegations_balance, escrow, total } =
-    //   account as Account
-    // return (
-    //   available === '0' &&
-    //   debonding_delegations_balance === '0' &&
-    //   delegations_balance === '0' &&
-    //   escrow === '0' &&
-    //   total === '0' &&
-    //   nonce === 0 &&
-    //   size === 'XXS'
-    // )
-    // TODO: we should also check the number of transactions, where it becomes available
+    const {
```

Comment:
Issue? Can't find the designated burn address anymore
https://explorer.dev.oasis.io/search?q=oasis1qzq8u7xs328puu2jy524w3fygzs63rv3u5967970
https://pr-1785.oasis-explorer.pages.dev/search?q=oasis1qzq8u7xs328puu2jy524w3fygzs63rv3u5967970

| Before | After |
| --- | --- |
|  ![explorer dev oasis io_search_q=oasis1qzq8u7xs328puu2jy524w3fygzs63rv3u5967970](https://github.com/user-attachments/assets/123cfe2e-da8b-4bd6-8b6f-cad0a0a80422)      |      ![pr-1785 oasis-explorer pages dev_search_q=oasis1qzq8u7xs328puu2jy524w3fygzs63rv3u5967970](https://github.com/user-attachments/assets/a1048dd5-9073-4a3f-ace3-e60dde4d2edf)  |

Reply:
This  might be a special case for Nexus. We could add a workaround on either backend or frontend...
(Backend preferable)

Reply:
Frontend fix but I would rather abandon and ignore: https://github.com/oasisprotocol/explorer/pull/1812